variables.tf

```diff
@@ -14,6 +14,24 @@ variable "instance_name" {
   description = "Name of the security and compliance instance that will be provisioned by this module"
 }
 
+variable "provider_type_instance_name" {
+  type        = string
+  description = "The name for the provider_type instance"
+  default     = null
+}
+
+variable "attributes" {
+  type        = map(string)
+  description = "The attributes for connecting to the provider type instance"
+  default     = {}
+}
```

Comment:
don't expose this as a variable, hard code it in the code using the value of `wp_instance_crn`